include/z64save.h

```diff
@@ -622,7 +622,7 @@ typedef enum {
 #define WEEKEVENTREG_09_20 PACK_WEEKEVENTREG_FLAG(9, 0x20)
 #define WEEKEVENTREG_09_40 PACK_WEEKEVENTREG_FLAG(9, 0x40)
 #define WEEKEVENTREG_09_80 PACK_WEEKEVENTREG_FLAG(9, 0x80)
-#define WEEKEVENTREG_10_01 PACK_WEEKEVENTREG_FLAG(10, 0x01)
+#define WEEKEVENTREG_TALKED_TO_TINGLE PACK_WEEKEVENTREG_FLAG(10, 0x01)
```

Comment:
```suggestion
#define WEEKEVENTREG_TALKED_TINGLE PACK_WEEKEVENTREG_FLAG(10, 0x01)
```
Seems like other weekeventregs don't used `TO` just `TALKED`, same for the others.